packages/elements/src/heatmap/helpers/text.ts

```diff
@@ -95,22 +95,12 @@ const getMaximumTextWidth = (ctx: CanvasRenderingContext2D, cells: HeatmapCell[]
 };
 
 /**
- * Calculate max length text for render Axis size
+ * Get text with maximum length to render Axis size
  * @param labels array string text for calculate
- * @returns max length text
+ * @returns Text with maximum length
  */
 const getMaximumLabelTextWidth = (labels: string[]): string => {
-  let maxLengthText = '';
-
-  for (let i = 0; i < labels.length; i++) {
-    if (maxLengthText.length >= labels[i].length) {
-      maxLengthText = maxLengthText;
-    }
-    else {
-      maxLengthText = labels[i];
-    }
-  }
-  return maxLengthText;
+  return labels.reduce((previousText, currentText) => previousText.length > currentText.length ? previousText : currentText);
```

Comment:
I'd say rename var from `xxText` to `xxLabel` would be better.
And `@returns Text with maximum length` -> `@returns Longest label in the array`

Btw, method name could be better :)

Reply:
Agreed and fixed.